app/openlxp_xms_project/settings.py

```diff
@@ -199,39 +197,36 @@
 }
 TECHNICAL_CONTACT = {
     "givenName": "Tech Person",
-    "emailAddress": "technical@localhost.com"
+    "emailAddress": "technical@localhost.com",
 }
 SUPPORT_CONTACT = {
     "givenName": "Support Person",
     "emailAddress": "support@localhost.com",
 }
-USER_ATTRIBUTES = [
-    "user_permanent_id",
-    "email",
-    "first_name",
-    "last_name"
-]
+USER_ATTRIBUTES = ["user_permanent_id", "email", "first_name", "last_name"]
 
 AUTHENTICATION_BACKENDS = (
-    'django.contrib.auth.backends.ModelBackend',
-    'openlxp_authentication.models.SAMLDBAuth',
+    "django.contrib.auth.backends.ModelBackend",
+    "openlxp_authentication.models.SAMLDBAuth",
 )
 
 REST_FRAMEWORK = {
-    'DEFAULT_AUTHENTICATION_CLASSES': [
-        'rest_framework.authentication.SessionAuthentication',
+    "DEFAULT_AUTHENTICATION_CLASSES": [
+        "rest_framework.authentication.SessionAuthentication",
+    ],
+    "DEFAULT_PERMISSION_CLASSES": [
+        "users.models.PermissionsChecker",
     ],
-    'DEFAULT_PERMISSION_CLASSES': [
-        'users.models.PermissionsChecker',
-    ]
 }
 
 OPEN_ENDPOINTS = [
-    '/api/auth/login',
-    '/api/auth/register',
-    '/api/auth/logout',
-    '/api/auth/validate',
-    '/api/catalogs/',
-    '/api/experiences/',
-    '/api/experiences/[a-z0-9]+/',
+    "/api/auth/login",
+    "/api/auth/register",
+    "/api/auth/logout",
+    "/api/auth/validate",
+    "/api/catalogs/",
+    "/api/catalogs/[a-zA-Z0-9]+/",
+    "/api/catalogs/[a-zA-Z0-9]+/[a-zA-Z0-9]+/",
+    "/api/experiences/",

```

Comment:
/api/experiences/ endpoint is no longer used, so both should be removed from OPEN_ENDPOINTS

Reply:
On it

Reply:
experience ids have - so "/api/catalogs/[a-zA-Z0-9]+/[a-zA-Z0-9]+/" should be updated to "/api/catalogs/[a-zA-Z0-9]+/[a-zA-Z0-9\-]+/" (added \-)

Reply:
We adjusted the ids to use metadata_hash_id's this keeps it consistent with the frontend structure.